docs/API.md

```
@@ -231,14 +310,15 @@ Allows players to offer or accept a draw agreement in PvP mode.
 *   **Request Body:**
     ```json
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)